### PR TITLE
Dynamic javascript containers

### PR DESCRIPTION
**Pull request by Kanyorok**
The branch contains javascript functionality for the dynamic speakers section in the home page.
- Added the speakers in a dynamic approach using Javascript.
- Added README AND License file.
- Added event listeners for reloading pages upon page resize.

### CODE REVIEW

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @Kanyorok , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
- Project deployed✔
- Contain Home page and About Page✔
- Mobile first✔

## Required Changes ♻️
_Check the comments under the review._
- [ ] 🙏 Please ensure that each text in your header uses the correct color and font to match the original design 🎉. Remember, the color and fonts are specified in the original design. If you are unable to obtain the correct background for the large text, you can simply use the provided color, which is #EC5242, for the text. This will enhance the appearance 🌹 of your website and make it as cool as the original design.

Your design:
image
image


Original design:
image

- [ ] 🙏Please change the style of the Main program section by following these instructions📜:
  - Ensure that you use the correct background. If you are unable to obtain the exact background, you can simply use the dark background color provided in the original design.
  - Increase the size of your icon to match the size in the original design and make it identical.
  Your design:
image

  Original design:
image

-[ ]🙏Please use the correct color for the Invited Guest section to match the original design, and add a horizontal line between the texts as it appeared in the original design. This will enhance the appearance 🌹 of your website and make it as cool as the original design.

Your design:
image

Original design:
image

- 🙏Please adjust the style for the logo's title line to be shorter than the text and also change the title color to white, as used in the original design. This will enhance the appearance🌹 of this section.
Your design:
image

Original design:
image

- Please follow the design on the footer where there are two parts of the text: the bottom part and the top part.
Your design:
image

Original design:
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -683,21 +780,21 @@ body {
...
-    color: rgb(244 94 44);
+    color: #ec5242;
   }
 
   .header h1 {
     font-weight: bold;
     font-family: 'Lato', sans-serif;
     font-size: 5em;
-    background: -webkit-linear-gradient(rgb(244 94 44), #333);
+    background: -webkit-linear-gradient(#ec5242, #333);
```
Comment (TadesseDev) [inline style.css:790]: - [Optional] Good job adding the color @Kanyorok, I notice you used the `Lato` font for the main heading. but according to the original design, the main heading seems to be `Cocogoose`. I would suggest checking into this.

File: style.css [hunk 2]
```diff
@@ -515,31 +687,37 @@ body {
...
 
   .sessions i {
-    margin-bottom: 16px;
+    margin-bottom: 35px;
     color: #ff5438;
+    font-size: 58px;
   }
 
   .sessions {
     display: flex;
```
Comment (TadesseDev) [inline style.css:696]: - [Optional] Dear @Kanyorok, Good job applying the right background as requested earlier. But I notice your `sessions` text is a lot bolder than the original design, As frontend engineers we need to deliver something identical to the original design. Please take a look at it. 

### Your design 
image
### Original design 
image

File: javascript/index.js [hunk 3]
```diff
@@ -11,10 +63,114 @@ window.addEventListener('DOMContentLoaded', () => {
...
+    return divCard;
+  };
+
+  function refreshPage() {
+    const winSize = window.innerWidth;
+    if (winSize >= 768) {
+      window.location.reload();
+    } else if (winSize < 768) {
+      window.location.reload();
+    }
```
Comment (TadesseDev) [inline javascript/index.js:137]: - [Optional] Dear @Kanyorok, I see you have used ` window.location.reload();` to adjust your page, and it reloads every time I switch from one screen size to the other. In general, this is not a good practice. (`IF POSSIBLE NEVER USE window.location.reload();`). We need to be careful on how much we update the DOM.

File: about.html [hunk 4]
```diff
@@ -50,6 +50,7 @@
                 <section class="header_about">
                     <h2>Sustainable Travel</h2>
                     <h1>ECO-FRIENDLY TRAVEL OPTIONS</h1>
+                    <hr/>
                     <div class="descriptor_about">
                         <p>promotes outdoor activities that encourage a deeper connection with nature, such as hiking, wildlife safaris, snorkeling, and nature photography. Highlight the importance of responsible behavior, including respecting wildlife habitats, leaving no trace, and following designated trails</p>
```
Comment (TadesseDev) [inline about.html:55]: - [Optional] Dear @Kanyorok, You did a reat job updating the background image for the desktop design as requested earlier, But we need to follow the design guideline including the background image for the mobile version too. Please consider updating your mobile design to match the original design. 
### Your design.
image

### Original design 
image

**Review by TadesseDev — APPROVED** [review]
### Hi @Kanyorok 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
240590909-9d125ce4-1a1e-4269-91d9-a07846cecea3




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**